tests/integration/records/test_api_literature.py

```diff
@@ -74,14 +148,44 @@ def test_literature_create_with_invalid_data(base_app, db, create_pidstore):
     record_control_number = str(data["control_number"])
 
     with pytest.raises(ValidationError):
-        record = LiteratureRecord.create(data)
+        LiteratureRecord.create(data)
 
     record_pid = PersistentIdentifier.query.filter_by(
         pid_value=record_control_number
     ).one_or_none()
     assert record_pid is None
 
 
+def test_literature_create_with_invalid_data_and_mutliple_pids(
+    base_app, db, create_pidstore
+):
+    doi_value = faker.doi()
+    arxiv_value = faker.arxiv()
+    data = {"arxiv_eprints": [{"value": arxiv_value}], "dois": [{"value": doi_value}]}
+    data = faker.record(with_control_number=True, data=data)
+    data["invalid_key"] = "should throw an error"
+    pid_lit_value = str(data["control_number"])
+    pid_arxiv_value = arxiv_value
+    pid_doi_value = doi_value
+
+    with pytest.raises(ValidationError):
+        LiteratureRecord.create(data)
+
+    record_lit_pid = PersistentIdentifier.query.filter_by(
+        pid_value=pid_lit_value
+    ).one_or_none()
+    record_arxiv_pid = PersistentIdentifier.query.filter_by(
+        pid_value=pid_arxiv_value
+    ).one_or_none()
+    record_doi_pid = PersistentIdentifier.query.filter_by(
+        pid_value=pid_doi_value
+    ).one_or_none()
+
+    assert record_lit_pid is None
+    assert record_arxiv_pid is None
+    assert record_doi_pid is None
+
+
```

Comment:
Can we add a test for duplicated PIDs?

Reply:
We have already here https://github.com/inspirehep/inspirehep/pull/2/files#diff-eb7f0308380f84c1972fe05952b996a7R95 and added for ``arxiv`` and ``doi`` individually